src/test/java/cz/codecamp/logger/loggers/JSONLoggerTest.java

```diff
@@ -0,0 +1,34 @@
+package cz.codecamp.logger.loggers;
+
+import cz.codecamp.logger.LogLevelEnum;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by Lenovo on 15.10.2016.
+ */
+public class JSONLoggerTest {
+    private JSONLogger jsnlgr;
+    @Before
+    public void setUp() throws Exception {
+        jsnlgr = new JSONLogger();
```

Comment:
nepoužívat zkratky - jednou se z toho zblázníš... a hlavně to nikdo jiný po tobě nepřečte ;)